packages/sitecore-jss-angular/src/components/default-empty-text-field-editing-placeholder.component.ts

```diff
@@ -4,7 +4,7 @@ import { Component } from '@angular/core';
  * Default component that will be rendered in pages when field is empty; applies for text, richtext, date and link fields.
  */
 @Component({
-  selector: 'sc-default-empty-text-field-editing-placeholder',
-  template: '<span>[No text in field]</span>',
+  selector: '<span>[sc-default-empty-text-field-editing-placeholder]</span>',
```

Comment:
Is it possible to render selector by using this approach?
How it is rendered in HTML?

Reply:
Yes it is.
![image](https://github.com/user-attachments/assets/76a0078d-4bf9-41b3-986a-531fee1e8b24)

Reply:
Cool, this looks much better!